relay-server/src/services/store.rs

```diff
@@ -1305,6 +1381,35 @@ struct SpanKafkaMessage<'a> {
     platform: Cow<'a, str>, // We only use this for logging for now
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+struct LogKafkaMessage<'a> {
+    #[serde(default)]
+    organization_id: u64,
+    #[serde(default)]
+    project_id: u64,
+    #[serde(default)]
+    timestamp_nanos: u64,
+    #[serde(default)]
+    observed_timestamp_nanos: u64,
+    #[serde(default)]
+    retention_days: u16,
+    #[serde(default)]
+    received: u64,
+    body: &'a RawValue,
+
+    trace_id: EventId,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    span_id: Option<&'a str>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    severity_text: Option<&'a str>,
```

Comment:
`severity_text` might at some point contain JSON-encoded characters, in which case parsing to `&str` will fail.

```suggestion
    #[serde(borrow, default, skip_serializing_if = "Option::is_none")]
    severity_text: Option<Cow<'a, str>>,
```